lib/api/units/validateStatusCode.js

```diff
@@ -0,0 +1,30 @@
+const { TextDiff } = require('../../validators/text-diff');
+
+const APIARY_STATUS_CODE_TYPE = 'text/vnd.apiary.status-code';
+
+function normalizeStatusCode(statusCode) {
+  return String(statusCode).trim();
+}
+
+/**
+ * Validates given real and expected status codes.
+ * @param {number} realStatusCode
+ * @param {number} expectedStatusCode
+ */
+function validateStatusCode(realStatusCode, expectedStatusCode) {
+  const validator = new TextDiff(
+    normalizeStatusCode(realStatusCode),
+    normalizeStatusCode(expectedStatusCode)
+  );
```

Comment:
**Sigh:** I wish this could be just something like `realStatusCode === expectedStatusCode` 😄 But let's keep all the `TextDiff` heavy lifting for now, this needs to be done step by step. What you did is right.

Reply:
Agree. At first glance this can be done with a simple weak comparison.
Noted: https://github.com/apiaryio/gavel.js/issues/159